deploy/10_cloud-ingress-operator-credentials-aws.CredentialsRequest.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
-  name: cloud-ingress-operator-credentials
+  name: cloud-ingress-operator-credentials-aws
   namespace: openshift-cloud-ingress-operator
 spec:
   secretRef:
-    name: cloud-ingress-operator-credentials
+    name: cloud-ingress-operator-credentials-aws
```

Comment:
How do we expect this to work in terms of migrating to a new version? Presumably we'll have an old `CredentialsRequest`/`Secret` hanging around (named `cloud-ingress-operator-credentials`) in the AWS clusters? Do we need to do cleanup in this PR or otherwise?

Reply:
I was thinking we clean it up in [managed-cluster-config](https://github.com/openshift/managed-cluster-config).  I'd prefer that kind of (temporary?) cruft go there instead of polluting operator code.

Reply:
Opened [OSD-5792](https://issues.redhat.com/browse/OSD-5792) for this part.